test/users.js

```diff
@@ -298,3 +298,90 @@ describe('/users GET', () => {
       });
   });
 });
+
+describe('/admin/users POST', () => {
+  it('should create user admin successfuly', done => {
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({
+        email: 'federico.casares@wolox.com.ar',
+        password: '12345678'
+      })
+      .then(response => {
+        chai
+          .request(server)
+          .post('/admin/users')
+          .set('authorization', `Bearer ${response.body.token}`)
+          .send({
+            name: 'Juan Ignacio',
+            surname: 'Molina',
+            email: 'juanignacio.molina@wolox.com.ar',
+            password: 'pussy123'
+          })
+          .end((err, res) => {
+            expect(err).to.be.null;
+            expect(res).to.have.status(200);
+            expect(res).to.be.a.json;
```

Comment:
we should also check that there is actually being created a new user in the db and not only the endpoint response (I missed this comment in previous PRs), otherwise the controller could have the succesful response wired and we are never creating new users and the tests never fail.

Reply:
I'm gonna fix all tests on another pr.

Reply:
Done in #16